apps/percona/workload/run_litmus_test.yml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: percona-loadgen 
+  namespace: litmus
+spec:
+  template:
+    metadata:
+      name: percona-loadgen
+      namespace: litmus
```

Comment:
@shashank855 Can we have labels defined for litmus job as well?

Reply:
@dargasudarshan sure